Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM centos:8
 
 ARG USER_ID=14
 ARG GROUP_ID=50
 
 MAINTAINER Fer Uria <fauria@gmail.com>
-LABEL Description="vsftpd Docker image based on Centos 7. Supports passive mode and virtual users." \
+LABEL Description="vsftpd Docker image based on Centos 8. Supports passive mode and virtual users." \
 	License="Apache License 2.0" \
 	Usage="docker run -d -p [HOST PORT NUMBER]:21 -v [HOST FTP HOME]:/home/vsftpd fauria/vsftpd" \
 	Version="1.0"
 
 RUN yum -y update && yum clean all
```

Comment:
I think it is not needed to update repositories. The image will be ~100MB lighter

Reply:
Probably best to have the latest updates, but that's not really the focus of this PR.